src/Ocelot/DownstreamUrlCreator/Middleware/DownstreamUrlCreatorMiddleware.cs

```diff
@@ -102,20 +98,25 @@ public async Task Invoke(HttpContext httpContext)
             await _next.Invoke(httpContext);
         }
 
-        private static void RemoveQueryStringParametersThatHaveBeenUsedInTemplate(DownstreamRequest downstreamRequest, List<PlaceholderNameAndValue> templatePlaceholderNameAndValues)
+        private static void RemoveQueryStringParametersThatHaveBeenUsedInTemplate(
+            DownstreamRequest downstreamRequest, List<PlaceholderNameAndValue> templatePlaceholderNameAndValues)
```

Comment:
Why was that changed? 😉 
There is no need to change! No new code!

Reply:
Fixed in commit https://github.com/ThreeMammals/Ocelot/commit/3c7781e7ceccf0a54e1feae91a07c78e12bcfa3d ✔️